### ISSUE TITLE
How to train the model?

### ISSUE THREAD

**Issue by Guodong0712** [cvqluu/TDNN#1]
Hi
Thank you in advance, 
I am trying to implement the X-vector using your TDNN model, but when I try to train it and the loss can not down and the rate of right is so low.
Thank you, could you give me some suggestions?
my input:  24-dim MFCC

                                                                                                                                        Yours sincerely
                                                                                                                                                 Ma

**Comment by qinxiaoyi** [comment]
> Hi
> Thank you in advance,
> I am trying to implement the X-vector using your TDNN model, but when I try to train it and the loss can not down and the rate of right is so low.
> Thank you, could you give me some suggestions?
> my input: 24-dim MFCC
> 
> ```
>                                                                                                                                     Yours sincerely
>                                                                                                                                              Ma
> ```

i have the same question, when i train the model, the loss always nan, than i exchange the relu and batchnorm, but the eer is very high

**Comment by Guodong0712** [comment]
Thanks for your reply, I am a freshman and have not to compute the eer because the final exam.

**Comment by Guodong0712** [comment]
> > Hi
> > Thank you in advance,
> > I am trying to implement the X-vector using your TDNN model, but when I try to train it and the loss can not down and the rate of right is so low.
> > Thank you, could you give me some suggestions?
> > my input: 24-dim MFCC
> > ```
> >                                                                                                                                     Yours sincerely
> >                                                                                                                                              Ma
> > ```
> 
> i have the same question, when i train the model, the loss always nan, than i exchange the relu and batchnorm, but the eer is very high

Hi，thank you first, I make the score with cosine, and about 0.998 between the same speaker utt and 
0.993 between the different, so I predict that eer will be high, can you give me some suggestions?

Ma

**Comment by HappyGallery** [comment]
> 
> 
> > Hi
> > Thank you in advance,
> > I am trying to implement the X-vector using your TDNN model, but when I try to train it and the loss can not down and the rate of right is so low.
> > Thank you, could you give me some suggestions?
> > my input: 24-dim MFCC
> > ```
> >                                                                                                                                     Yours sincerely
> >                                                                                                                                              Ma
> > ```
> 
> i have the same question, when i train the model, the loss always nan, than i exchange the relu and batchnorm, but the eer is very high

Hi ，if you have solved the question ,loss  always nan .I have meet the problem  too  , but i have no method to solve  . if you know how please teach me . Thanks!

**Comment by HappyGallery** [comment]
@qinxiaoyi
